senders/slack/slack.go

```diff
@@ -89,15 +90,32 @@ func (sender *Sender) buildMessage(events moira.NotificationEvents, trigger moir
 		message.WriteString(trigger.Name)
 	}
 
+	messageCharsCount := len([]rune(message.String()))
+
 	if trigger.Desc != "" {
 		message.WriteString("\n")
-		message.WriteString(trigger.Desc)
+		charsRequiredForEvents := 9000
+		charsAvailableForDesc := messageMaxCharacters - messageCharsCount - charsRequiredForEvents - 1
+
+		// Replace **bold text** with *bold text* that slack supports
+		re := regexp.MustCompile(`(?m)\*\*(?P<boldtext>[\w\s]+)\*\*`)
```

Comment:
Once we got our application started:
- how many times sender.SendEvents() will be executed?
- and how many times sender.Init() will be executed?
:)